composer.json

```diff
@@ -7,15 +7,20 @@
     "license": "Apache-2.0",
     "require": {
         "php": ">=5.4",
-        "google/auth": "dev-master"
+        "google/auth": "dev-master",
+        "monolog/monolog": "^1.17",
+        "phpseclib/phpseclib": "~2.0"
     },
     "require-dev": {
         "phpunit/phpunit": "3.7.*",
         "squizlabs/php_codesniffer": "~2.3"
     },
     "autoload": {
+        "psr-0": {
+            "Google_": "src/"
+        },
         "classmap": [
-            "src/"
+            "src/Google/Service/"
         ]
```

Comment:
Sorry Michael! I saw "mt" and my brain filled in the blanks. 

We will make it a priority to switch to Guzzle 6 before tagging this release. Thanks for reaching out!